### PR TITLE
Adding jupyter_resource_usage_proxy

### PR DESCRIPTION
**Pull request by benmoss**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/jupyter_resource_usage_proxy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_resource_usage_proxy-{{ version }}.tar.gz
+  sha256: 5d474bd3c78271d6924c4c202da89da01a1a3c176027796513567f950c6e92ab
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```
Comment (jakirkham) [inline recipes/jupyter_resource_usage_proxy/meta.yaml:14]: Conda-Build configures these for us in the build. So think we can simplify like so

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/jupyter_resource_usage_proxy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_resource_usage_proxy-{{ version }}.tar.gz
+  sha256: 5d474bd3c78271d6924c4c202da89da01a1a3c176027796513567f950c6e92ab
+
+build:
```
Comment (jakirkham) [inline recipes/jupyter_resource_usage_proxy/meta.yaml:12]: Personally like to put this at the top of the `build` section to make it easier to get to when doing updates. Though that is just personal preference

```suggestion
build:
  number: 0
```

File: recipes/jupyter_resource_usage_proxy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
```
Comment (jakirkham) [inline recipes/jupyter_resource_usage_proxy/meta.yaml:15]: Goes with the other `number` comment

```suggestion
```

File: recipes/jupyter_resource_usage_proxy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    - jupyter_server >=1.0.0
+    - httpx >=0.23.1
+
+test:
+  imports:
+    - jupyter_resource_usage_proxy
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (jakirkham) [inline recipes/jupyter_resource_usage_proxy/meta.yaml:33]: Personally preference. Though please do whatever is easier

```suggestion
test:
  requires:
    - pip
  imports:
    - jupyter_resource_usage_proxy
  commands:
    - pip check
```

**Review by jakirkham — COMMENTED** [review]
Thanks Ben! 🙏

Couple minor suggestions below

File: recipes/jupyter_resource_usage_proxy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+about:
+  summary: A Jupyter Server Extension Proxying Resource Usage.
+  license: MIT
+  license_file: LICENSE
+  home: https://github.com/davidbrochart/jupyter-resource-usage-proxy
+
+extra:
+  recipe-maintainers:
+    - benmoss
```
Comment (davidbrochart) [inline recipes/jupyter_resource_usage_proxy/meta.yaml:43]:
```suggestion
    - benmoss
    - davidbrochart
```

**Comment by benmoss** [comment]
Thanks @jakirkham and @davidbrochart  😄

**Comment by jakirkham** [comment]
Thanks Ben & David! 🙏

**Pull request merged**